Gruntfile.js

```diff
@@ -282,6 +282,8 @@ module.exports = function(grunt) {
       options: {
         urls: ["http://127.0.0.1:9999/test/tests.html"],
         testname: "Plottable Sauce Unit Tests",
+        pollInterval: 5000,
+        statusCheckAttempts: 60,
```

Comment:
Total time is = pollInterval * statusCheckAttempts. Since our test suite is a single page, it does not make much difference. 

Another reasoning I went through was that polling less often means disturbing the test less often, hence the system might run faster. 

The downside of that is that tests will finish in a second that is multiple of 5, so if a test finishes in 41 seconds, it will only be acknowledged at second 45